editor/utils/tracking.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
Then later we can move it over to https://github.com/WordPress/packages (I thought that was what you meant).